Riot/Modules/Room/ContextualMenu/ReactionsMenu/ReactionsMenuViewModel.swift

```diff
@@ -0,0 +1,202 @@
+/*
+ Copyright 2019 New Vector Ltd
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+
+final class ReactionsMenuViewModel: ReactionsMenuViewModelType {
+
+    // MARK: - Properties
+
+    // MARK: Private
+    private let aggregations: MXAggregations
+    private let roomId: String
+    private let eventId: String
+
+    // MARK: Public
+
+    var isAgreeButtonSelected: Bool = false
```

Comment:
Use `private(set) var` for this property and 3 others below to make them mutable only in this class.